mobile/lib/modules/asset_viewer/views/gallery_viewer.dart

```diff
@@ -52,15 +51,15 @@ class GalleryViewerPage extends HookConsumerWidget {
       [],
     );
 
-    @override
-    initState(int index) {
-      indexOfAsset = index;
-    }
-
     getAssetExif() async {
-      assetDetail = await ref
-          .watch(assetServiceProvider)
-          .getAssetById(assetList[indexOfAsset].id);
+      if (assetList[indexOfAsset.value].isRemote) {
+        assetDetail = await ref
+            .watch(assetServiceProvider)
+            .getAssetById(assetList[indexOfAsset.value].id);
+      } else {
+        // TODO local exif parsing?
```

Comment:
This would be amazing, though I don't know if it should only happen when the user click the more info button since we will have to open and parse the file through an EXIF parser

Reply:
There are basically two options:

1. When finding a new local asset on the device, always load the original file and parse the exif and store the info in some local state. -> only makes sense if we actually keep a local state (the current cache is insufficient!) in a database. Still wasteful if the user never looks at the details and the images are uploaded to the server and parsed again.
2. Only load the file and parse exif if the user actually requests that information. -> might take a few milliseconds but should be fast enough to almost be unnoticeable.

We will decide an what to do in a follow-up PR :-)